libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -45,6 +45,9 @@
         [iconTitle]="title"
         (addOnButtonClicked)="_addOnButtonClicked()"
     >
+        <span [attr.id]="tokenHiddenId" aria-hidden="true" class="fd-multi-input__invisible-text">
+            Contains {{ selected.length || 'no' }} token</span
```

Comment:
I think we should add i18n support here

Reply:
fixed

Reply:
We're not using `@angular/localize` in `core`, use `@Input()` instead for passing aria labels.

Reply:
Please use `@Input()` so the customers can pass translations for this string.